src/uct/ib/rc/base/rc_iface.h

```diff
@@ -98,6 +98,7 @@ enum {
 
 /* flags for uct_rc_iface_send_op_t */
 enum {
+    UCT_RC_IFACE_SEND_OP_STATUS     = UCS_BIT(11), /* status field is valid */
```

Comment:
do we really need to add this flag? maybe only as assert flag?

Reply:
this is needed not only as a assert flag, it is needed as a functional flag